discovery-infra/tests/base_cancel_reset.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pytest
+
+import assisted_service_api
+import utils
+import consts
+
+if os.environ.get('NODE_ENV') == 'QE_VM':
```

Comment:
You made the same condition inside `node_controller` fixture. You can remove it from here

Reply:
done